vignettes/containerit.Rmd

```diff
@@ -70,177 +134,202 @@ session <- clean_session(exp, echo = TRUE)
 dockerfile_object <- dockerfile(from = session)
 ```
 
-```{r}
+```{r, results='markup', comment=NA}
 print(dockerfile_object)
 ```
 
+### 2.4 Packaging an R script
 
-## 2.2 Packaging an R script
-
-R scripts are packaged by just supplying the file paths as a from-argument. They are automatically copied into the container's working directory. In order to run the R script on start-up, rather than an interactive R session, a CMD instruction can be supplied using `CMD_Rscript` path as an argument to `cmd`.
+R scripts are packaged by just supplying the file path or paths to the arguement `from` of `dockerfile()`.
+They are automatically copied into the container's working directory.
+In order to run the R script on start-up, rather than an interactive R session, a CMD instruction can be added by providing the value of the helper function `CMD_Rscript()` as an argument to `cmd`.
 
 ```{r}
-setwd(tempdir())
-scriptFile <- tempfile(pattern = "containeRit_", fileext = ".R")
+# create simple script file
+scriptFile <- tempfile(pattern = "containerit_", fileext = ".R")
 writeLines(c('library(rgdal)',
              'nc <- rgdal::readOGR(system.file("shapes/", package="maptools"), "sids", verbose = FALSE)',
              'proj4string(nc) <- CRS("+proj=longlat +datum=NAD27")',
              'plot(nc)'), scriptFile)
 
-dockerfile_object <- dockerfile(from = scriptFile, silent = TRUE, cmd = CMD_Rscript(scriptFile))
-unlink(scriptFile)
-print(dockerfile_object)
-```
-
+# use a custom startup command
+scriptCmd <- CMD_Rscript(scriptFile)
```

Comment:
This, I think is what we talked about formerly:
should fix with scriptCmd <- CMD_Rscript(basename(scriptFile))